include/uapi/linux/rpmsg.h

```diff
@@ -24,4 +24,7 @@ struct rpmsg_endpoint_info {
 #define RPMSG_CREATE_EPT_IOCTL	_IOW(0xb5, 0x1, struct rpmsg_endpoint_info)
 #define RPMSG_DESTROY_EPT_IOCTL	_IO(0xb5, 0x2)
 
+/* The feature bitmap for virtio rpmsg */
+#define VIRTIO_RPMSG_F_NS	0 /* RP supports name service notifications */
```

Comment:
@lyakh : Curious to know why this line went into this file, instead of virtio_rpmsg.h?

Reply:
@ymdatta it's a part of the UAPI and all other feature bits are also defined under uapi/linux e.g. include/uapi/linux/virtio_net.h, include/uapi/linux/virtio_iommu.h etc.